light/graph/builders/one_room_builder.py

```diff
@@ -22,15 +23,19 @@
     DBEdgeType,
 )
 
-import os
 import random
 import time
-import asyncio
 
 from dataclasses import dataclass, field
 from omegaconf import MISSING, DictConfig  # type: ignore
 from typing import Dict, Any, List, Optional, Tuple, TYPE_CHECKING
 
+from light.registry.model_pool import  ModelTypeName
+
+from light.registry.model_pool import  ModelTypeName
+
+from light.registry.model_pool import  ModelTypeName
+
```

Comment:
Hm probably merge weirdness.

Reply:
Yeah, cleaned up the imports here a bit.